about.html

```diff
@@ -0,0 +1,90 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>About ME Section</title>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.1/css/all.min.css"
+        integrity="sha512-MV7K8+y+gLIBoVD59lQIYicR65iaqukzvf/nwasF0nqhPay5w/9lJmVM2hMDcnK1OnMGCdVK+iQrJ7lzPJQd1w=="
+        crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <link rel="stylesheet" href="about.css">
+</head>
+<body>
+    <main>
+       <header>
+            <nav>
+           <a class="about-logo" href="./index.html"><img src="./images/CC_Global_Summit_2015 logo.png" alt="img"></a>
+           <ul class="desktop-ul">
+           <li class="desktop-link"><a href="./about.html" class="nav-bar">About</a></li>
+           <li class="desktop-link"><a href="#">Program</a></li>
+           <li class="desktop-link"><a href="#">Join</a></li>
+           <li class="desktop-link"><a href="#">Sponser</a></li>
+           <li class="desktop-link"><a href="#">News</a></li>
+           <li class="desktop-link"><a href="#" class="nav">CC Compaign</a></li>
+           </ul>
+               <div class="menu-icon">
+                   <div class="item-closed"><a href="#" class="closing"><i class="fa fa-times" aria-hidden="true"></i></a>
+                   </div>
+                   <div class="mobile-nav"><a href="./index.html" class="closing">Home</a></div>
+                   <div class="mobile-nav"><a href="./about.html" class="closing">About</a></div>
+                   <div class="mobile-nav"><a href="#" class="closing">Join</a></div>
+                 </div>
+           <div class="item-menu"><a href="#" class="openMenu"><i class="fa fa-bars"></i></a></div>
+           </nav>
+           </header>
+        <section class="about-me">   
```

Comment:
- [ ] Please make sure your about page headline section elements have the same `font size`, `color`, `height`, `width`, and spacing `padding/spacing` as defined in the original design. For example, your headline title needs to be `uppercase`. Please note that is very important to stick to the original design as much as possible :+1: 

Check the screenshots below to compare and see the differences :point_down: 

####  Original design
![Screenshot from 2022-10-28 18-44-06](https://user-images.githubusercontent.com/6615394/198710820-dbee4473-7cbe-49b6-bc50-ab59c70c4a8b.jpg)